scanner/src/main/kotlin/ScanResultsStorage.kt

```diff
@@ -103,6 +103,11 @@ abstract class ScanResultsStorage {
             val readers = config.storageReaders.orEmpty().map { resolve(it) }
             val writers = config.storageWriters.orEmpty().map { resolve(it) }
 
+            log.info {
```

Comment:
Do we really need this as we already have similar output in [ScannerCommand](https://github.com/oss-review-toolkit/ort/blob/b43f388e58d90105f5c827917e10ede9a812d16d/cli/src/main/kotlin/commands/ScannerCommand.kt#L128)?

Reply:
But this output does only show the name of the storage, nothing about the configuration.